src/ripple/app/ledger/Ledger.cpp

```diff
@@ -1762,7 +1694,9 @@ void Ledger::updateFees ()
     }
 
     {
-        StaticScopedLockType sl (sPendingSaveLock);
+        // VFALCO Why not do this before calling getASNode?
+        std::lock_guard<
+            std::mutex> lock(mutex_);
```

Comment:
Or if you're really itching to get the deduction today:
````
auto lock = beast::make_lock(mutex_);
````
from https://github.com/ripple/rippled/blob/develop/src/beast/beast/utility/make_lock.h#L31